types/discord.ts

```diff
@@ -99,6 +99,8 @@ export interface DiscordConnection {
 
   /** An array of partial server integrations */
   integrations?: DiscordIntegration[];
+  /** Whether this connection has a corresponding third party OAuth2 token. */
+  two_way_link: boolean;
```

Comment:
Does this need a change in a transformer?

Reply:
No, we currently do not have a transformer for that type.